Frechet_distance/include/CGAL/Frechet_distance.h

```diff
@@ -27,6 +28,61 @@
 namespace CGAL
 {
 
+#ifdef CGAL_EIGEN3_ENABLED
+template<class Dim> struct Epick_d;
+template<class Dim> struct Epeck_d;
+
+template < typename P, int D>
+struct Ambient_dimension<P, Epick_d<Dimension_tag<D>>>
```

Comment:
Should that go into `Epick_d.h`    @sloriot  ?

Reply:
@mglisse ?

Reply:
I guess we could add a template Ambient_dimension inside the class Kernel_d_interface that says everything has Base::Dimension as ambient dim, since that's what CGAL::Ambient_dimension looks for by default.